server/api/events/add.post.ts

```diff
@@ -0,0 +1,30 @@
+import { PrismaClient } from "@prisma/client";
+
+export default defineEventHandler(async event => {
+  try {
+    const prisma = new PrismaClient();
+    const { name, location, summary, description, slides, organizer, startTime, endTime } =
+      await readBody(event);
```

Comment:
are we not doing any validation on this here?
https://github.com/wobsoriano/h3-zod would come in very handy...